src/test/java/org/eclipse/yasson/customization/JsonbNillableTest.java

```diff
@@ -87,6 +85,7 @@ public void testNillableInConfig() {
     
     public static class PrimitiveNullBoolean {
 
+        @SuppressWarnings("deprecation")
```

Comment:
What exactly does this mean on the field?

Reply:
We are using the deprecated parameter (nillable) on the annotation (@JsonbProperty) here. Therefore, I have a suppressor on the field. I think this is an old test from the times when this parameter wasn't deprecated. So suppression is ok right now; this test would be removed as soon as the parameter would be also removed and wouldn't be just deprecated.